.env

```
@@ -4,6 +4,13 @@ APP_SECRET=2e222659c5b006df8d0e68cc33913706
 API_BASE_URL=https://localhost
 API_URL=https://api.vilnacrm.com
 API_PREFIX=/api
+# Trust only the directly-connected reverse proxy (Caddy/FrankenPHP, AWS App
+# Runner load balancer). REMOTE_ADDR trusts a single hop so getClientIp()
+# returns the real client IP for IP-keyed rate limiters and X-Forwarded-For
+# from untrusted clients is ignored. In production override TRUSTED_PROXIES
+# with the exact proxy CIDR. Only X-Forwarded-For is trusted.
+TRUSTED_PROXIES=REMOTE_ADDR
+TRUSTED_HEADERS=x-forwarded-for
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRUSTED_HEADERS key should go before the TRUSTED_PROXIES key

(UnorderedKey)